avaframe/tests/test_getStats.py

```diff
@@ -22,27 +23,27 @@ def test_getStats(tmp_path):
 
     # define test directory
     avaName = 'avaStats'
-    avaDirtmp = os.path.join(tmp_path, avaName)
-    avaDirPeakFiles = os.path.join(tmp_path, avaName, 'Outputs', 'com1DFA', 'peakFiles')
-    avaDirConfigFiles = os.path.join(tmp_path, avaName, 'Outputs', 'com1DFA', 'configurationFiles')
+    avaDirtmp = pathlib.Path(tmp_path, avaName)
+    avaDirPeakFiles = avaDirtmp / 'Outputs' / 'com1DFA' / 'peakFiles'
+    avaDirConfigFiles = avaDirtmp / 'Outputs' / 'com1DFA' / 'configurationFiles'
     os.makedirs(avaDirtmp)
     os.makedirs(avaDirPeakFiles)
     os.makedirs(avaDirConfigFiles)
 
     # set path to required input files
-    dirPath = os.path.dirname(__file__)
-    testDataDir = os.path.join(dirPath, 'data')
-    data1 = os.path.join(testDataDir, 'testGetStats_1000000_ppr.asc')
-    data2 = os.path.join(testDataDir, 'testGetStats_2000000_ppr.asc')
-    cfg1 = os.path.join(testDataDir, 'testGetStats_1000000.ini')
-    cfg2 = os.path.join(testDataDir, 'testGetStats_2000000.ini')
+    dirPath = pathlib.Path(__file__).parents[0]
+    testDataDir = pathlib.Path(dirPath, 'data')
+    data1 = testDataDir / 'testGetStats_1000000_ppr.asc'
+    data2 = testDataDir / 'testGetStats_2000000_ppr.asc'
+    cfg1 = testDataDir / 'testGetStats_1000000.ini'
+    cfg2 = testDataDir / 'testGetStats_2000000.ini'
 
     # define destination paths
-    fNames = [os.path.join(avaDirPeakFiles, 'release1_null_dfa_1000000_ppr.asc'),
-              os.path.join(avaDirPeakFiles, 'release1_null_dfa_2000000_ppr.asc')]
+    fNames = [str(avaDirPeakFiles / 'release1_null_dfa_1000000_ppr.asc'),
+              str(avaDirPeakFiles / 'release1_null_dfa_2000000_ppr.asc')]
 
-    configFiles = [os.path.join(avaDirConfigFiles, 'release1_null_dfa_1000000.ini'),
-                   os.path.join(avaDirConfigFiles, 'release1_null_dfa_2000000.ini')]
+    configFiles = [str(avaDirConfigFiles / 'release1_null_dfa_1000000.ini'),
+                   str(avaDirConfigFiles / 'release1_null_dfa_2000000.ini')]
 
     # copy files
     shutil.copy(data1, fNames[0])
```

Comment:
ŵhy do you need to copy it?

Reply:
I added a comment - because the avaDir folder structure is required by extractMAxValues